### DISCUSSION TITLE
How does publishing a site currently work with flowershow?

### DISCUSSION THREAD

**Discussion by ski-freak** [flowershow/flowershow#555] [General and Help]
What is my best option if I just want to host markdown files somewhere for free? Does flowershow support that? Or do I need to pay for my own hosting elsewhere?

**Comment by danielo515** [comment]
There is nothing free. There are always options with free tier, but even in that case you will at least need a domain, if you don't want one of the ugly free domains this servoces offer. Vercel has currently a free tier, doesn't that work for you?

**Reply by nosovk** [reply]
You can use: 
https://vercel.com/
https://pages.cloudflare.com/
https://www.netlify.com/

all of them have some free tier, and all of them can host nextjs site (flowershow is a nextjs based site)
